.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         name: Generate Alpine
         shell: bash
         run: |
-          echo "::set-output name=versions::[\"3.14\", \"3.13\", \"3.12\", \"3.11\", \"3.10\"]"
+          echo "::set-output name=versions::[\"3.14\", \"3.13\", \"3.12\", \"3.11\"]"
```

Comment:
It's important to mention that we are no longer building the images, but they will still be available at https://hub.docker.com/r/usabillabv/php/tags?page=1&ordering=last_updated&name=3.10 :)